aptos-move/e2e-testsuite/src/tests/verify_txn.rs

```diff
@@ -1089,3 +1092,17 @@ fn test_type_tag_transitive_dependency_fails_verification() {
         _ => panic!("Kept transaction with an invariant violation!"),
     }
 }
+
+pub fn encode_test_coin_transfer(to: AccountAddress, amount: u64) -> TransactionPayload {
```

Comment:
we should add these to a manually written aptos_stdlib ... say aptos_stdlib_extensions

Reply:
Updated